src/App.tsx

```diff
@@ -14,11 +15,151 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    selectedGood: '',
+    selectedGoods: [],
+  };
 
-export default App;
+  render() {
+    const {
+      selectedGood, selectedGoods,
+    } = this.state;
+
+    const message = () => {
```

Comment:
```suggestion
    const getMessage = () => {
```

Reply:
fix it - use as name what method will do